Formula/svgbob.rb

```diff
@@ -0,0 +1,35 @@
+class Svgbob < Formula
+  desc "Convert your ascii diagram scribbles into happy little SVG"
+  homepage "https://ivanceras.github.io/svgbob-editor/"
+  url "https://github.com/ivanceras/svgbob/archive/0.5.0-alpha.6.tar.gz"
```

Comment:
I'd request a stable version upstream, cause I'd really want to leave the allowlist alone

Reply:
Ok, I'll close this for now and let the upstream know that formula is ready to be reused once they cut the stable release 🤝